.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,701 @@
+
+   ![Fork button on GitHub](../assets/images/fork_button_github.png)
+
+2. Choose your username in the 'Owner *' dropdown menu. **Don't select an organization here. Always choose your personal account**; otherwise, maintainers can't collaborate and fix things for you on your PR.
+
+   ![Choose fork owner on GitHub](../assets/images/choose_fork_owner_github.png)
+
+3. Uncheck the option of 'Copy the DEFAULT-BRANCH-NAME branch only' because later on, you want to be able to clone multiple branches.
+
+4. Click the green 'Create fork' button at the bottom.
+
+   ![An uncheck state option to copy only default branch and a create fork button on GitHub](../assets/images/uncheck_option_and_create_fork_button_github.png)
+
+### Clone the repository
+
+After you forked the repository, you need to clone it. Cloning means making a copy of a repository in your local environment. In this case, you want to clone your forked repository.
+
+> [!IMPORTANT]
+>
+> The Mautic User Documentation contains multiple branches that represent specific versions of Mautic. You should clone each branch into its own dedicated folder and make your changes within the appropriate folder.
+>
+> So, for example, when you need to make documentation changes for Mautic versions 7 and 6, clone the branch `7.0` and save it in a folder, then clone the branch `6.0` and save it in a separate folder. You can name the folder anything you want, but it's best to reflect the branch name. For example, use names like `user-docs-7`, `user-docs-6`, and so on.
+
+Follow the steps below to clone your forked repository:
+
+1. Click your avatar on the top right.
+2. Click 'Repositories'.
+
+   ![Repositories option from a dropdown menu on GitHub](../assets/images/repositories_option_github.png)
+
+3. Open your forked user-documentation repository. The URL should have your username. For example: `https://github.com/YOUR-GITHUB-USERNAME/user-documentation`.
+4. Click the green 'Code' button on top.
+5. Select 'HTTPS' and copy the URL.
+
+   ![Highlight of code button, copy symbol, and HTTPS tab on GitHub](../assets/images/code_button_https_tab_github.png)
+
+6. In your terminal, go to your local directory where you want to save this project.
+7. Run the `git clone` command specifying the branch and folder name, and hit Enter:
+
+   ```bash
+   git clone --branch BRANCH-NAME https://github.com/YOUR-GITHUB-USERNAME/user-documentation FOLDER-NAME
+   ```
+
+   Here's an example:
+
+   ```bash
+   git clone --branch 7.0 https://github.com/YOUR-GITHUB-USERNAME/user-documentation user-docs-7
+   ```
+
+### Create a new branch
+
+Before making changes, ensure that you create a new branch and work on it. You don't want to directly work on the default — such as `main` — or any other base branch, because you won't be able to work on lots of things at the same time. If you make all those changes on one branch, it's not possible to separate them out and only merge one change at a time.
```

Comment:
```suggestion
Before making changes, ensure that you create a new branch and work on it. You don't want to directly work on the default - such as `main` - or any other base branch, because you won't be able to work on lots of things at the same time. If you make all those changes on one branch, it's not possible to separate them out and only merge one change at a time.
```